packages/core-components/src/components/HeaderIconLinkRow/IconLinkVertical.tsx

```diff
@@ -29,6 +29,7 @@ export type IconLinkVerticalProps = {
   label: string;
   onClick?: React.MouseEventHandler<HTMLAnchorElement>;
   title?: string;
+  linkProps?: LinkProps;
```

Comment:
Instead of adding a new `linkProps` you can just change the `IconLinkVerticalProps` to be an interface extending `LinkProps`, e.g. like here: https://github.com/backstage/backstage/blob/master/packages/core-components/src/components/SimpleStepper/SimpleStepperFooter.tsx#L45-L49

Make sure to omit props you don't want to override (e.g. `to` & `className` as they already are existing props). The `interface` should not require any other props than currently as that could break existin usages of the component. Other props can be removed (like e.g. `onClick` or `title`) as they are already part of `LinkProps`. The remaining props can than be passed through using the spread operator - like you did bellow.

Reply:
Hi, thanks for your feedbacks again!

Updated my PR :+1: